src/client/interpreter/contracts.ts

```diff
@@ -37,7 +37,10 @@ export interface IComponentAdapter {
     hasInterpreters: Promise<boolean | undefined>;
     getInterpreters(resource?: Uri): Promise<PythonEnvironment[] | undefined>;
     // WorkspaceVirtualEnvInterpretersAutoSelectionRule
-    getWorkspaceVirtualEnvInterpreters(resource: Uri): Promise<PythonEnvironment[] | undefined>;
+    getWorkspaceVirtualEnvInterpreters(
+        resource: Uri,
+        options?: GetInterpreterLocatorOptions,
```

Comment:
We don't need all of `GetInterpreterLocatorOptions`.

```suggestion
        options?: { ignoreCache: boolean },
```